src/GraphQLParser.ApiTests/GraphQLParser.approved.txt

```diff
@@ -852,7 +856,11 @@ namespace GraphQLParser.Visitors
     public interface IPrintContext : GraphQLParser.Visitors.IASTVisitorContext
     {
         int IndentLevel { get; set; }
+        bool IndentPrinted { get; set; }
```

Comment:
@Shane32 Formally, this is breaking changes, but I doubt that there are a sufficient number of people for whom it will be a real problem